README.md

```diff
@@ -142,3 +142,19 @@ namespace MyNamespace
     }
 }
 ```
+
+If you want to use an external font, include the external font file as an additional file in your csproj:
+
+```xml
+<ItemGroup>
+    <AdditionalFiles Include="myfont.flf" />
+</ItemGroup>
+```
+
+Then specify the font name in the attribute:
+
+```c#
+[GenerateFiggleText("HelloWorldString", "myfont", "Hello world")]
+```
+
+Note the font name is case-sensitive and must match the name in the font file.
```

Comment:
What do you think about making the lookup case-insensitive? If we're looping through additional files, it'd be easy enough to use `StringComparer.OrdinalIgnoreCase.Equals` for the comparison.

Reply:
Yes I agree case-insensitive is more friendlier.  I've updated to use `StringComparison.OrdinalIgnoreCase` when comparing the font's file name.